src/Components/CreateImageWizard/formComponents/Oscap.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from 'react';
```

Comment:
Should we add an expandable with the name of the selected policy to the Review step? We can also add it in a follow up PR

Reply:
I think that this is a good idea! I'll add a commit for that.